CocoaMarkdown/CMNodeKey.h

```diff
@@ -0,0 +1,24 @@
+//
+//  CMNodeKey.h
+//  CocoaMarkdown
+//
+//  Created by Anton Barkov on 8/13/19.
+//  Copyright © 2019 Indragie Karunaratne. All rights reserved.
```

Comment:
Is this ok that file created by you, but copyrighted to owner of CocoaMarkdown?

Reply:
I'll leave it this way for now, we have similar copyrights in new files in third party frameworks.

Reply:
👌